.storybook/config/assignement-contexts.js

```diff
@@ -0,0 +1,113 @@
+    hideCode: "Hide code",
+    unlockedStateAction: "Continue",
+    lockedStateAction: "Unlock",
+  },
+  params: {
+    startSceneId: 1,
+    scenes: [
+      {
+        sceneId: 1,
+        scenename: "test",
+        interactions: [
+          {
+            label: {
+              labelText: "svar riktig",
+              labelPosition: "right",
+            },
+            action: {
+              library: "H5P.SingleChoiceSet 1.11",
+              params: {  },
+              metadata: {},
+            },
+            interactionpos: "50,0",
+          },
+          {
+            "interactionpos": "50,50",
+            label: {
+              labelText: "svar riktig",
+              labelPosition: "right",
+            },
+            "action": {
+                "library": "H5P.Summary 1.10",
+                "params": {
+                    "intro": "<p>Hvor mange bilder er det p\u00e5 veggen?<\/p>\n",
+                    "summaries": [
+                        {
+                            "subContentId": "8442f194-ff46-4a49-9b5e-bb82f8ba4269",
+                            "summary": [
+                                "<p>1<\/p>\n",
+                                "<p>2<\/p>\n",
+                                "<p>3<\/p>\n"
+                            ],
+                            "tip": ""
+                        }
+                    ],
+                    "overallFeedback": [
+                        {
+                            "from": 0,
+                            "to": 99,
+                            "feedback": "Feil."
+                        },
+                        {
+                            "from": 100,
+                            "to": 100,
+                            "feedback": "Korrekt. Det er 3 bilder. Her er koden: 2534"
+                        }
+                    ],
+                    "solvedLabel": "Progress:",
+                    "scoreLabel": "Wrong answers:",
+                    "resultLabel": "Your result",
+                    "labelCorrect": "Correct.",
+                    "labelIncorrect": "Incorrect! Please try again.",
+                    "alternativeIncorrectLabel": "Incorrect",
+                    "labelCorrectAnswers": "Correct answers.",
+                    "tipButtonLabel": "Show tip",
+                    "scoreBarLabel": "You got :num out of :total points",
+                    "progressText": "Progress :num of :total"
+                },
+                "subContentId": "fe51c9c0-d9a2-41b1-ab45-45936718793c",
+                "metadata": {
+                    "contentType": "Summary",
+                    "license": "U",
+                    "title": "Untitled Summary"
+                }
+            }
+          }
+        ],
+      },
+    ],
+  },
+  contentId: 0,
+  extras: {},
+  sceneRenderingQuality: "high",
+  on: () => undefined,
+  getRatio: () => 16 / 9,
+  trigger: (eventName, data) => {
+    console.log(eventName);
+    console.log(data);
```

Comment:
Use `console.info(...params)` if the logs are supposed to be here. This way, a quick `console.log` search will reveal any deletable dev logs.
```suggestion
```
```suggestion
    console.info(eventName, data);
```